.golangci.yml

```diff
@@ -45,9 +45,27 @@ linters:
     - nonamedreturns # Consider to enable it
     - nlreturn # Consider to enable it
     - ireturn  # Consider to enable it (??)
-    - dupword  # Consider to enable it
 
 linters-settings:
+  dupword:
```

Comment:
Please revert the config file

Reply:
Do you mean I should remove the `dupword` and its setting from `linters-settings` ?

Reply:
keep `dupword` but remove its settings